dali/c_api/c_api.h

```diff
@@ -26,6 +26,11 @@ extern "C" {
     void* ws;
   };
 
+  enum device_type_t {
```

Comment:
Maybe we could move this enum to `dali/plugin/copy.h`, so we can also use it in `dali/python/backend_impl.cc`

Reply:
We can't as I want to have that in C API so TF plugin can use it. We can copy it to the C++ header as well, but I don't know if we should duplicate that.

Reply:
Just made copy-paste to the copy.h.